### DISCUSSION TITLE
How to run the panel?

### DISCUSSION THREAD

**Discussion by agn-7** [AdguardTeam/AdGuardHome#5216] [Q&A] [answered]
I installed the app on linux and ran it via `sudo /opt/AdGuardHome/AdGuardHome -s start`. But how can I see the panel GUI?!

**Comment by ghost** [comment, marked as answer]
When you run that command from the cli you'll find the address you need to access it from in the cli.

Find the instructions here also: https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#first-time
